koans/EmptyIsNullOrIssetKoans.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace PhpKoans;
+use PHPUnit\Framework\TestCase;
+defined('__') or define('__', null);
+
+class EmptyIsNullOrIssetKoans extends TestCase
+{
+    /**
+     * @testdox is_null return true if variable is null
+     */
+    function testMethodIsNull()
+    {
+        $this->assertEquals(__, is_null(NULL));
```

Comment:
If you want to use the variable null in upper case, use it everywhere the same, but don't use upper case first and lower case everywhere else.